src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -283,37 +282,32 @@ private static string CreateErrorDescription(Exception authFailure)
             {
                 // Order sensitive, some of these exceptions derive from others
```

Comment:
Is this comment still relevant? Does the switch select the most specific match in a type hierarchy?